spec/services/collators/gross_income_collator_spec.rb

```diff
@@ -51,48 +52,44 @@ module Collators
             end
 
             it "updates the gross income record with categorised monthly incomes" do
-              collator
-              gross_income_summary.reload
-              expect(gross_income_summary.benefits_all_sources).to be_zero
-              expect(gross_income_summary.maintenance_in_all_sources).to be_zero
-              expect(gross_income_summary.pension_all_sources).to be_zero
-              expect(gross_income_summary.friends_or_family_all_sources).to eq 105.13
-              expect(gross_income_summary.property_or_lodger_all_sources).to eq 66.45
-              expect(gross_income_summary.monthly_other_income).to eq 171.58
-              expect(gross_income_summary.total_gross_income).to eq 171.58
+              response = collator
+              expect(response.monthly_regular_incomes(:all_sources, :benefits)).to be_zero
+              expect(response.monthly_regular_incomes(:all_sources, :maintenance_in)).to be_zero
+              expect(response.monthly_regular_incomes(:all_sources, :pension)).to be_zero
+              expect(response.monthly_regular_incomes(:all_sources, :friends_or_family)).to eq 105.13
+              expect(response.monthly_regular_incomes(:all_sources, :property_or_lodger)).to eq 66.45
+              expect(response.total_gross_income).to eq 171.58
```

Comment:
have we lost the monthly_other_income check?

Reply:
Yup - turns out nothing other than tests was ever reading `monthly_other_income` so I've removed it.